ui/app/AppLayouts/Profile/panels/ProfileSocialLinksPanel.qml

```diff
@@ -0,0 +1,214 @@
+                color: Theme.palette.baseColor1
+                text: qsTr("Your links will appear here")
+            }
+
+            ShapePath {
+                fillColor: "transparent"
+                strokeColor: Theme.palette.baseColor2
+                strokeWidth: 1
+                strokeStyle: ShapePath.DashLine
+                dashPattern: [4, 4]
+
+                startX: shape.radius
+                startY: 0
+                PathLine {
+                    x: shape.width - shape.radius
+                    y: 0
+                }
+                PathCubic {
+                    control1X: shape.width
+                    control2X: shape.width
+                    control1Y: 0
+                    control2Y: 0
+                    x: shape.width
+                    y: shape.radius
+                }
+                PathLine {
+                    x: shape.width
+                    y: shape.height - shape.radius
+                }
+                PathCubic {
+                    control1X: shape.width
+                    control2X: shape.width
+                    control1Y: shape.height
+                    control2Y: shape.height
+                    x: shape.width - shape.radius
+                    y: shape.height
+                }
+                PathLine {
+                    x: shape.radius
+                    y: shape.height
+                }
+                PathCubic {
+                    control1X: 0
+                    control2X: 0
+                    control1Y: shape.height
+                    control2Y: shape.height
+                    x: 0
+                    y: shape.height - shape.radius
+                }
+                PathLine {
+                    x: 0
+                    y: shape.radius
+                }
+                PathCubic {
+                    control1X: 0
+                    control2X: 0
+                    control1Y: 0
+                    control2Y: 0
+                    x: shape.radius
+                    y: 0
+                }
+            }
+        }
+
+        StatusListView {
+            id: linksView
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            model: root.socialLinksModel
+
+            displaced: Transition {
+                NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
+            }
+
+            delegate: DropArea {
+                id: delegateRoot
+
+                property int visualIndex: index
+
+                width: ListView.view.width
+                height: draggableDelegate.height
+
+                keys: ["x-status-draggable-list-item-internal"]
+
+                onEntered: function(drag) {
+                    const from = drag.source.visualIndex
+                    const to = draggableDelegate.visualIndex
+                    if (to === from)
+                        return
+                    console.debug("!!! MOVE FROM", from, "TO", to)
```

Comment:
leftover?

Reply:
Well not really... it's a debug, so it won't appear in production builds; I can remove if you insist :)

Reply:
Just a question! Up to you! ;)

Reply:
Removed :)